tests/json_schema_test.go

```diff
@@ -165,103 +165,3 @@ func testDataAgainstSchema(t *testing.T, testData []schemaTestData, schemaPath s
 		assert.True(t, filesToTest.Has(f.Name()), fmt.Sprintf("Did you miss to add the file %v to `json_schema_tests`?", filepath.Join(path, f.Name())))
 	}
 }
-
-func TestGetSchemaProperties(t *testing.T) {
```

Comment:
This got removed because `flattenJsonKeys` is now doing the work that `flattenSchemaNames` used to do, but the JSON Spec part of `flattenJsonKeys` isn't tested as far as I can see.

Reply:
It is not, but i know it works because the tests using it pass.